core/libdeno/api.cc

```diff
@@ -198,6 +200,7 @@ void deno_respond(Deno* d_, void* user_data, deno_op_id op_id, deno_buf buf) {
   }
 
   auto v = recv_->Call(context, context->Global(), argc, args);
+  d->isolate_->RunMicrotasks();
```

Comment:
I think this is the same as kAuto... I benchmarked it and there's no change

master
```
~/src/deno> wrk http://127.0.0.1:4500/
Running 10s test @ http://127.0.0.1:4500/
  2 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   291.26us  494.34us  11.83ms   96.74%
    Req/Sec    20.33k     2.10k   32.32k    82.50%
  404493 requests in 10.00s, 19.67MB read
Requests/sec:  40447.21
Transfer/sec:      1.97MB
~/src/deno> wrk http://127.0.0.1:4500/
Running 10s test @ http://127.0.0.1:4500/
  2 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   272.21us  386.03us   6.74ms   97.01%
    Req/Sec    20.61k     1.59k   24.46k    68.50%
  410139 requests in 10.00s, 19.95MB read
Requests/sec:  41001.46
Transfer/sec:      1.99MB
```

this branch
```
~/src/deno> wrk http://127.0.0.1:4500/
Running 10s test @ http://127.0.0.1:4500/
  2 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   272.63us  402.47us   6.44ms   97.03%
    Req/Sec    20.81k     1.43k   28.58k    83.17%
  418073 requests in 10.10s, 20.33MB read
Requests/sec:  41393.01
Transfer/sec:      2.01MB
~/src/deno> wrk http://127.0.0.1:4500/
Running 10s test @ http://127.0.0.1:4500/
  2 threads and 10 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   271.93us  400.53us   6.75ms   97.05%
    Req/Sec    20.84k     1.22k   26.85k    80.69%
  418670 requests in 10.10s, 20.36MB read
Requests/sec:  41454.11
Transfer/sec:      2.02MB
```

Reply:
That's my experience as well, calling `run_microtasks` inside `Isolate::poll` gives about 10-15% less req/sec.